newrelic/example_test.go

```diff
@@ -1,25 +0,0 @@
-// Copyright 2019 New Relic Corporation. All rights reserved.
-// SPDX-License-Identifier: Apache-2.0
-
-package newrelic_test
-
-import (
-	"log"
-	"os"
-
-	"github.com/newrelic/opentelemetry-exporter-go/newrelic"
-	"go.opentelemetry.io/otel/api/global"
-	"go.opentelemetry.io/otel/sdk/trace"
-)
-
-func Example() {
-	exporter, err := newrelic.NewExporter("My Service", os.Getenv("NEW_RELIC_API_KEY"))
-	if err != nil {
-		log.Fatal(err)
-	}
-	tp, err := trace.NewProvider(trace.WithSyncer(exporter))
-	if err != nil {
-		log.Fatal(err)
-	}
-	global.SetTraceProvider(tp)
-}
```

Comment:
We should keep this example because it shows up in godocs.

Reply:
rolling back, should keep the examples folder as well?